lib/OpenQA/WebSockets/Controller/Worker.pm

```diff
@@ -224,10 +224,10 @@ sub _message {
             log_debug("Rescheduling jobs assigned to worker $worker_id");
             $worker->reschedule_assigned_jobs([$current_job, @unfinished_jobs]);
         }
-        catch {
+        catch ($e) {
```

Comment:
The `try` block above is quite long and has three `return` statements.
Please check if this is really working as intended. Tests might pass, but maybe just because this is not covered.

Reply:
If in doubt, leave that file out for now. Before doing any refactorings.